kogito-usertasks-custom-lifecycle-quarkus/README.md

```diff
@@ -12,18 +12,43 @@ This example shows
 
 * working with user tasks
 * four eye principle with user tasks
-	
-	
+
+In addition this quickstart also shows how to use custom life cycle for user tasks that uses custom life cycle phases compared to those supported by default.
+
+Start
+Complete - extended the default one that allows only to complete started tasks
+
+
 <p align="center"><img width=75% height=50% src="docs/images/process.png"></p>
+	
+Diagram Properties (top)
+<p align="center"><img src="docs/images/diagramProperties.png"></p>
+
+* Diagram Properties (bottom)
+<p align="center"><img src="docs/images/diagramProperties3.png"></p>
+
+* First Line Approval (top)
+<p align="center"><img src="docs/images/firstLineApprovalUserTask.png"></p>
 
+* First Line Approval (bottom)
+<p align="center"><img src="docs/images/firstLineApprovalUserTask2.png"></p>
+
+* First Line Approval (Assignments)
+<p align="center"><img src="docs/images/firstLineApprovalUserTaskAssignments.png"></p>
+
+* Second Line Approval
+<p align="center"><img src="docs/images/secondLineApprovalUserTask.png"></p>
+
+* Second Line Approval (Assignments)
+<p align="center"><img src="docs/images/secondLineApprovalUserTaskAssignments.png"></p>
 
 In addition this quickstart also shows how to use custom life cycle for user tasks that uses custom life cycle phases
```

Comment:
As we copied that sentence at the beginning of the document, you can remove it from there.

Reply:
fixed @radtriste

Reply:
fixed @radtriste